packages/element-lib/src/sidetree-v2/protocol/getTransactions.js

```diff
@@ -0,0 +1,37 @@
+const getTransactions = sidetree => async () => {
+  const transactions = await sidetree.blockchain.getTransactions(
+    0,
+    'latest',
+    { omitTimestamp: true },
+  );
+  // Only get the last 20 transactions to avoid crashing the page
```

Comment:
This comment does not belong in the library... The library is not only used by web pages.

Limits should happen in the app, not the library... the app should use the library to request a data slice that it can render, not rely on the library to decide this kind of thing.

Reply:
good point